JavaScript/JavaScriptSDK/RequestResponseHeaders.ts

```diff
@@ -5,17 +5,32 @@ module Microsoft.ApplicationInsights {
         /**
          * Request-Context header
          */
-        public static requestContextHeader = "request-context";
+        public static requestContextHeader = "Request-Context";
 
         /**
          * Target instrumentation header that is added to the response and retrieved by the
          * calling application when processing incoming responses.
          */
         public static requestContextTargetKey = "appId";
 
+        /**
+         * Request-Context appId format
+         */
+        public static requestContextAppIdFormat = "appId=cid-v1:";
+
         /**
          * Request-Id header
          */
         public static requestIdHeader = "Request-Id";
+
+        /**
+         * Sdk-Context header.
```

Comment:
Can you expand this comment explaining where this header is used?

Reply:
Done